src/EditorFeatures/Core/Implementation/CommentSelection/AbstractCommentSelectionBase.cs

```diff
@@ -37,6 +37,9 @@ internal enum Operation
 
     internal abstract class AbstractCommentSelectionBase<TCommand>
     {
+        protected const string LanguageNameString = "languagename";
```

Comment:
don't you have same thing defined in comment block? share the code?

Reply:
That's why I moved it, so I could reuse it :)  This is the same one being used in toggle block